proto/stat_var.proto

```diff
@@ -34,20 +34,20 @@ import "entity.proto";
 
 message PlaceStatVarExistence {
   // Set if this value is for a place+SVG-ID key, but not for place+SV-ID key.
-  int32 num_descendent_stat_vars = 1;
+  int64 num_descendent_stat_vars = 1;
 }
 
 message StatVarSummary {
   message Place {
     string dcid = 1;
-    string name = 2;
-    int64 person_count = 3;
+    string name = 3;
+    int64 person_count = 2;
```

Comment:
This is dropped now?

Reply:
https://source.corp.google.com/piper///depot/google3/datacommons/proto/prophet/cache_data.proto;dlc=426232895;bpv=0;bpt=0;rcl=427571961

Reply:
I will remove it when switch to the new cache, otherwise the Unmarshal would fail now.